index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <nav class="flex-toolbar">
+      <div class="logo-name"><p>Jesus Molina</p></div>
```

Comment:
- [ ] The logo design is great, but it should be a link to the home page and would be better wrapped in an `<a>` tag for the better convention, semantics, and user experience(UX). Kindly change that.

Reply:
Fixed